WhiteDentalClinic/WhiteDentalClinic.DataAccess/Repositories/DentistServiceRepository/DentistServiceRepositoryBase.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WhiteDentalClinic.DataAccess.Entities.CustomerEntity;
+using WhiteDentalClinic.DataAccess.Entities.ServiceDentist;
+using WhiteDentalClinic.DataAccess.Repositories.CustomerRepository;
+
+namespace WhiteDentalClinic.DataAccess.Repositories.DentistServiceRepository
+{
+    public class DentistServiceRepositoryBase : BaseRepository<DentistServiceEntity>, IDentistServiceRepository
```

Comment:
I'm a bit lost about this DentistServiceRepositoryBase as it already inherits some behavior from BaseRepository. Why would we need another base repo for dentists service?